projects/plugins/jetpack/modules/subscriptions/views.php

```diff
@@ -210,43 +210,37 @@ static function render_widget_status_messages( $instance ) {
 					break;
 				case 'spammed':
-					echo "<div style='background-color: #{$themecolors['bg']}; border: 1px solid #{$themecolors['border']}; color: #{$themecolors['text']}; padding-left: 5px; padding-right: 5px; margin-bottom: 10px;'>";
-					echo wp_kses_post( sprintf( __( 'Because there are many pending subscriptions for this email address, we have blocked the subscription. Please <a href="%s">activate or delete</a> pending subscriptions before attempting to subscribe.' ), 'https://subscribe.wordpress.com/' ) );
-					echo "</div>";
+					/* translators: %s is a URL */
+					$message = sprintf( __( 'Because there are many pending subscriptions for this email address, we have blocked the subscription. Please <a href="%s">activate or delete</a> pending subscriptions before attempting to subscribe.', 'jetpack' ), 'https://subscribe.wordpress.com/' );
 					break;
 				case 'subscribed':
-					echo "<div style='background-color: #{$themecolors['bg']}; border: 1px solid #{$themecolors['border']}; color: #{$themecolors['text']}; padding-left: 5px; padding-right: 5px; margin-bottom: 10px;'>";
-					_e( 'You&rsquo;re already subscribed to this site.' );
-					echo "</div>";
+					$message = __( 'You&rsquo;re already subscribed to this site.', 'jetpack' );
 					break;
 				case 'pending':
-					echo "<div style='background-color: #{$themecolors['bg']}; border: 1px solid #{$themecolors['border']}; color: #{$themecolors['text']}; padding-left: 5px; padding-right: 5px; margin-bottom: 10px;'>";
-					_e( 'You have a pending subscription already; we just sent you another email. Click the link or <a href="https://en.support.wordpress.com/contact/">contact us</a> if you don&rsquo;t receive it.' );
-					echo "</div>";
+					$message = __( 'You have a pending subscription already; we just sent you another email. Click the link or <a href="https://en.support.wordpress.com/contact/">contact us</a> if you don&rsquo;t receive it.', 'jetpack' );
 					break;
 				case 'confirmed':
-					echo "<div style='background-color: #{$themecolors['bg']}; border: 1px solid #{$themecolors['border']}; color: #{$themecolors['text']}; padding-left: 5px; padding-right: 5px; margin-bottom: 10px;'>";
-					_e( 'Congrats, you&rsquo;re subscribed! You&rsquo;ll get an email with the details of your subscription and an unsubscribe link.' );
-					echo "</div>";
+					$message = __( 'Congrats, you&rsquo;re subscribed! You&rsquo;ll get an email with the details of your subscription and an unsubscribe link.', 'jetpack' );
 					break;
 			}
+
+			$border_color = isset( $themecolors['border'] ) ? " #{$themecolors['border']}" : '';
+			echo sprintf( "<div style='border: 1px solid%s; padding-left: 5px; padding-right: 5px; margin-bottom: 10px;'>", esc_attr( $border_color ) );
+			echo wp_kses_post( $message );
+			echo '</div>';
```

Comment:
What would you think about simplifying things even more?

```suggestion
			printf(
				'<div style="border: 1px solid%1$s; padding-left: 5px; padding-right: 5px; margin-bottom: 10px;">%2$s</div>',
				esc_attr( $border_color ),
				wp_kses_post( $message )
			);
```

Reply:
Nice, even better. Committed!